conversation/ui
# Add new TC logos and icons

This PR adds new logo and icon image assets to the project (thanks @ZoeJazz)

They will be used on the republishing guidelines page.

## How to test

Run `make dist` and check the new image assets appear in the `dist/esm/logos` directory.
